src/main/java/uk/gov/cshr/service/RoleServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.cshr.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.cshr.domain.Role;
+import uk.gov.cshr.repository.RoleRepository;
+
+import java.util.Optional;
+
+@Service
+public class RoleServiceImpl implements RoleService {
```

Comment:
As mentioned this is redundant. But leave for now - I just wanted to put my objection in writing 😝